src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css

```diff
@@ -989,6 +989,10 @@
   line-height: 18px;
 }
 
+.ez-embed-type-image img {
```

Comment:
I wonder: `.ez-embed-type-image` is not enough, we need to apply this directly to `img` element?

Reply:
Apparently, it didn't work when applied to parent divs.

Reply:
What was the reason it didn't work?

I would expect a generic CSS rule for images, like `img { max-width: 100%; }`.
If something overrides it, then it has to be checked why and fixed.

Reply:
There is no generic rule for img. A really similar approach is used for rendering images in content/view: https://github.com/ezsystems/ezplatform-admin-ui/blob/1.2/src/bundle/Resources/public/scss/_general.scss#L42

![content_view](https://user-images.githubusercontent.com/34310128/45957072-6dded880-c014-11e8-899e-d83840e9f51d.png)

If you meant `img { max-width: 100%; }` set globally, it would work as it isn't overriden, but I am a bit unsure it it wouldn't ruin something else.